api/roles/rolesRouter.js

```diff
@@ -2,7 +2,7 @@ const express = require('express');
 const router = express.Router();
 const Roles = require('./rolesModel');
 
-const { validateUser } = require('../middleware/generalMiddleware');
+const { validateUser } = require('../middleware/validationMiddleware');
```

Comment:
This change was made to match the change in naming convention to the validationMiddleware.js file.